### ISSUE TITLE
Duplicated errors 

### ISSUE THREAD

**Issue by AndreyUtka** [s-panferov/awesome-typescript-loader#116]
After switching from ts-loader to awesome-typescript-loader, I noticed that if I've some error in my code - I'll see duplicated errors in console.

For example - https://github.com/AndreyUtka/angular-typescript-starter-kit - if you run it, and make some error: 

```typescript
export class HelloComponent implements ng.IComponentOptions {
    - displayName: string;
    + displayName: number;
    style: any;

    static selector = "hello";
    static template = require("./hello.html");
    static controller = HelloComponent;
    static $inject = ["$element"];

    constructor($element) { }

    $onInit() {
        this.displayName = "firstName";
        this.style = require("./hello.css");
    }

    $onChanges(object: any) {
        console.warn(object);
    }

    $onDestroy() {
        console.warn("on destroy");
    }
}

``` 

terminal show duplicated errors. With the same configuration with ts-loader - everything is ok

awesome-typescript-loader:

![2016-04-06_19445](https://cloud.githubusercontent.com/assets/1304912/14325049/26b84644-fc30-11e5-86e3-394fc6e2137b.png)

ts-loader: 

![2016-04-06_1944](https://cloud.githubusercontent.com/assets/1304912/14325055/317a2700-fc30-11e5-9ec2-0127f4c8a25d.png)

**Comment by s-panferov** [comment]
@AndreyUtka hello, sorry for the delay with answer. I'll check this.

**Comment by AndreyUtka** [comment]
@s-panferov after small investigation I find that it happens in 'after-compile' event in webpack compile lifecycle. I tried to skip this process:

```typescript
    compiler.plugin('after-compile', function(compilation, callback) {
        if (compilation.compiler.isChild()) {
            callback();
            return;
        }
        ...
```
and it helps.

**Comment by s-panferov** [comment]
Should be fixed in v0.17.0.

**Comment by AndreyUtka** [comment]
thanks a lot)  I checked it in my project. everything is ok 👍
